repose-aggregator/components/filters/header-translation-filter/src/main/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilter.scala

```diff
@@ -65,22 +65,29 @@ class HeaderTranslationFilter @Inject()(configurationService: ConfigurationServi
 
       sourceHeaders foreach { sourceHeader =>
         val originalHeaderName = sourceHeader.getOriginalName
-        val originalHeaderValues = httpRequest.getHeadersScala(originalHeaderName)
+        val originalHeaderValues = if (sourceHeader.isSplittable)
+          httpRequest.getSplittableHeaderScala(originalHeaderName) else
+          httpRequest.getHeadersScala(originalHeaderName)
```

Comment:
Oh, was it one line that got split by the IntelliJ formatter? That would make sense.

One line is correct, but due to the length you may prefer:
```
if (sourceHeader.isSplittable)
    httpRequest.getSplittableHeaderScala(originalHeaderName)
else
    httpRequest.getHeadersScala(originalHeaderName)
```